### DISCUSSION TITLE
how can i add prometheus alert rules?

### DISCUSSION THREAD

**Discussion by zhuw0312** [prometheus-operator/kube-prometheus#1479] [Q&A]
This repository now has the new GitHub Discussions enabled: 
https://github.com/coreos/kube-prometheus/discussions

Please create a new discussion to ask for any kind of support, which is not a Bug or Feature Request.

Thank you for being part of this community!

---

We are still happy to chat with you in the #prometheus-operator channel on Kubernetes Slack!

**Comment by paulfantom** [comment]
There is a doc about it https://github.com/prometheus-operator/kube-prometheus/blob/main/docs/developing-prometheus-rules-and-grafana-dashboards.md#prometheus-rules
